extension/README.md.args.mjs

```diff
@@ -0,0 +1,250 @@
+export const skipQuickStart = true;
+
+export const extraContents = `# 🚩 Challenge 2: 🏵 Token Vendor 🤖
+
+![readme-2](https://github.com/scaffold-eth/se-2-challenges/assets/55535804/b427171f-3f20-41a5-b76f-05d67e2b9959)
+
+🤖 Smart contracts are kind of like "always on" _vending machines_ that **anyone** can access. Let's make a decentralized, digital currency. Then, let's build an unstoppable vending machine that will buy and sell the currency. We'll learn about the "approve" pattern for ERC20s and how contract to contract interactions work.
+
+🏵 Create \`YourToken.sol\` smart contract that inherits the **ERC20** token standard from OpenZeppelin. Set your token to \`_mint()\` **1000** (\* 10 \*\* 18) tokens to the \`msg.sender\`. Then create a \`Vendor.sol\` contract that sells your token using a payable \`buyTokens()\` function.
+
+🎛 Edit the frontend that invites the user to input an amount of tokens they want to buy. We'll display a preview of the amount of ETH it will cost with a confirm button.
+
+🔍 It will be important to verify your token's source code in the block explorer after you deploy. Supporters will want to be sure that it has a fixed supply and you can't just mint more.
+
+🌟 The final deliverable is an app that lets users purchase your ERC20 token, transfer it, and sell it back to the vendor. Deploy your contracts on your public chain of choice and then \`yarn vercel\` your app to a public web server. Submit the url on [SpeedRunEthereum.com](https://speedrunethereum.com)!
+
+> 💬 Meet other builders working on this challenge and get help in the [Challenge 2 Telegram](https://t.me/joinchat/IfARhZFc5bfPwpjq)!
+
+---
+
+## Checkpoint 0: 📦 Environment 📚
```

Comment:
@rin-st You forgot to remove Checkpoint 0.

Reply:
we don't need to remove Checkpoint 0, we need just these changes inside checkpoint 0.
```
: node/yarn/git and npx create-eth parts are removed since we don't need those parts from inside an already created instance. Also yarn chain/deploy/start is a bit different.
```

Sorry if it's not clear in checklist

Reply:
ok, thanks!!